dts/riscv/it8xxx2.dtsi

```diff
@@ -708,6 +708,20 @@
 			pinctrl-0 = <&pinctrl_i2c_clk3    /* GPH1 */
 				     &pinctrl_i2c_data3>; /* GPH2 */
 		};
+		i2c3_1: i2c@f03681 {
+			compatible = "ite,it8xxx2-i2c";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x00f03680 0x0080>;
+			interrupts = <4 IRQ_TYPE_LEVEL_HIGH>;
+			interrupt-parent = <&intc>;
+			status = "disabled";
+			label = "I2C_3_1";
+			port-num = <3>;
+			gpio-dev = <&gpiof>;
```

Comment:
As a follow up PR, can you extract the gpio-dev directly from the pinctrl-0 property? 
```
struct i2c_alts_cfg {
	/* Pinmux control group */
	const struct device *pinctrls;
	/* GPIO device */
	const struct device *gpio_dev;
	/* GPIO pin */
	uint8_t pin;
	/* Alternate function */
	uint8_t alt_fun;
};
```

Reply:
The gpio-dev controller is needed in the i2c node for the recovery bus, not all drviers need gpio-dev. Is it necessary to add it to the pinctrl-0 property?